api/src/main/java/jakarta/data/page/PageRequest.java

```diff
@@ -77,7 +77,7 @@ public interface Pageable<T> {
      * typed {@link Sort} instances. For example,</p>
      *
      * <pre>
-     * {@code Pageable<Car>} page1Request = Pageable.of(Car.class).page(1).size(25).sortBy(
+     * {@code PageRequest<Car>} page1Request = PageRequest.of(Car.class).page(1).size(25).sortBy(
      *                                          Sort.desc("price"),
      *                                          Sort.asc("mileage"),
      *                                          Sort.asc("vin"));
```

Comment:
Make the indentation align after the rename:
```suggestion
     * {@code PageRequest<Car>} page1Request = PageRequest.of(Car.class).page(1).size(25).sortBy(
     *                                                Sort.desc("price"),
     *                                                Sort.asc("mileage"),
     *                                                Sort.asc("vin"));
```